src/i18n/es.json

```diff
@@ -0,0 +1,137 @@
+﻿{
+  "common": {
+    "cancel": "Cancelar",
+    "copyright_MD": "© {year} Doppler LLC. Todos los derechos reservados. [Política de Privacidad y Legales](https://www.fromdoppler.com/legales/privacidad).",
+    "help": "Ayuda",
+    "hide": "Ocultar",
+    "message": "Mensaje",
+    "recaptcha_legal_HTML": "Sitio protegido por reCAPTCHA. <a href=\"https://policies.google.com/privacy?hl=es\">Política de Privacidad</a> y <a href=\"https://policies.google.com/terms?hl=es\">Condiciones del Servicio</a> de Google.",
+    "send": "Enviar",
+    "show": "Mostrar"
+  },
+  "feature_panel": {
+    "email_automation": "Email automation",
```

Comment:
`Email Automation`

Reply:
@carodipietro esto que comenta @cbernat no es parte de los cambios de @GusBaamonde, pero me parece que los comentarios de Cecilia son correctos, ¿Crees que deberíamos aplicar esta sugerencia?

Reply:
@cbernat is right.